docs/src/pages/layout/GuttersLayout.js

```diff
@@ -1,8 +1,8 @@
 // @flow weak
-
 import React, { Component } from 'react';
 import { createStyleSheet } from 'jss-theme-reactor';
 import customPropTypes from 'material-ui/utils/customPropTypes';
+import type { Gutters } from 'material-ui/Layout/Layout';
```

Comment:
That's a demo, I'm not sure we want any typing stuff related inside them. The demo won't work out of the box for people not using flow. There is a  lot including me.